exec.go

```diff
@@ -225,11 +225,17 @@ func execute(context *cli.Context) error {
 		return err
 	}
 
+	user := fmt.Sprintf("%d:%d", params.ociProcess.User.UID, params.ociProcess.User.GID)
+
+	if params.ociProcess.User.Username != "" {
+		user = params.ociProcess.User.Username
```

Comment:
Are you sure we want to default to this in case we don't have anything provided. I think we should force `0:0`. This should be cleaner than to expect this `Username` to be 0 in order to set the user as root.

Reply:
I'm not sure I understand - the code will default the the UID+GID provided by the OCI config file, but if the user specifies `--user=`, we'll honour that instead. But note that the `--user=` case only applies for running `cc-runtime` "standalone" and has no impact when run under docker.

Reply:
Oh yes I get it. I have misread the code...